app/Enums/PetActivityLevel.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Enums;
+
+enum PetActivityLevel: String
```

Comment:
PHP doesn't have `String` type. Pleas use `string`.

Maybe it's a good thing to implement into our codestyle package?